### ISSUE TITLE
Prediction accuracy/probability (OCR)

### ISSUE THREAD

**Issue by fedExpress** [ria-com/nomeroff-net#88]
Добрый день.
Такой вопрос: каким образом можно расчитать общий результат вероятности? Как я понял, score = массив прогнозирований. Каким образом возможно рассчитать точность/уверенность используя данные значения?
textDetector = TextDetector({
    "eu_ua_2004_2015": {
        "for_regions": ["eu_ua_2015", "eu_ua_2004"],
        "model_path": "latest"
    },
    "eu": {
        "for_regions": ["eu", "eu_ua_1995"],
        "model_path": "latest"
    },
    "ru": {
        "for_regions": ["ru", "eu-ua-fake-lnr", "eu-ua-fake-dnr"],
        "model_path": "latest"
    },
    "kz": {
        "for_regions": ["kz"],
        "model_path": "latest"
    },
    "ge": {
        "for_regions": ["ge"],
        "model_path": "latest"
    }
})
im = cv2.imread(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'images\\image.png'))
text_arr, score = textDetector.predict([im], ["eu"], [1], return_acc=True)
@dimabendera

**Comment by dimabendera** [comment]
Это можно сделать следующим образом
```python
textArr, ocr_score = textDetector.predict(zones, regionNames, lines, return_acc=True)
text_acc_orig      = textDetector.get_acc(ocr_score, textArr, regionNames)
```

Для нахождения вероятности используется ctc function, хотелось бы заметить что это ресурсоёмкая операция

**Comment by arudakov** [comment]
разобрался - делюсь решением
в OCR.py 438 строка меняем с 
return  1 - loss.eval(session=K.get_session())
на
return 1 - loss.eval(session=tf.compat.v1.keras.backend.get_session())

в demo3 в секцию import добавляем

import tensorflow as tf
tf.compat.v1.disable_eager_execution()

![accuracy](https://user-images.githubusercontent.com/34522310/98244003-c9b01580-1f7f-11eb-8882-ee581dcc5f5c.png)

**Comment by ApelSYN** [comment]
@arudakov, спасибо за решение.
